synapse/cores/common.py

```diff
@@ -663,60 +631,55 @@ def _synTagsVers0(self):
                 self.delRowsByProp(oldp)
                 self.delRowsByProp('_:dark:tag', valu=tag)
 
-    def _initCoreMods(self):
-        # call our interal model revision functions
-        revs = [
-            (0, self._synTagsVers0),
-        ]
-        self.revModlVers('syn:core', revs)
```

Comment:
This removed running _synTagsVers0 and seting the syn:core model to rev0

Reply:
yep.  it still gets set below.

Reply:
remove this key